### DISCUSSION TITLE
Video instead of Timelapse Photo

### DISCUSSION THREAD

**Discussion by motionben** [gopro/labs#342] [Q&A] [answered]
Hello everyone,

I tried to set up my GoPro 8 Black with GoPro Labs, but when I'm using the timelapse with some additional parameters/actions it always starts recording video instead of making pictures.

The parameters/actions I'm using are: "mTP,!12:00S,!17:00E,/timeR"

Basically I need to make a timelapse over a very long time (60days) and I want it to take a photo each hour starting from 12:00 and ending at let's say 17:00. Then it should repeat on the next day, when a test run works well, I will connect it with a powerbank.

The Gopro starts exactly at that time, but it starts recording a video instead of taking just one photo.

Anyone had the same problem?

Thank you so much for your support,
Benjamin

**Comment by dnewman-gpsw** [comment]
Have you create a Photo Timelapse preset?   mTP will select the correct presets, but only if it exists, and it doesn't by default on any HERO camera.  

Info from https://gopro.github.io/labs/control/custom/
`
Time/Night-Lapse modes: The camera defaults to having presets for only one time-lapse mode and one night-lapse mode, yet both modes have two configurations (video or photo.) For all QR Code mode commands to work, a preset is needed for every mode you wish to use. e.g. If you only have a preset for night-lapse video (NLV), the QR command for NLV will work, but will not for night-lapse photo (NLP). To solve this, create a new preset for NLP, and both video and photo night-lapse commands will work.`

**Reply by motionben** [reply]
Thank you @dnewman-gpsw
I checked the Gopro in Timelapse mode and there is a Photo Timelapse Preset. I created a new one with the mentioned custom link, but it's always recording a video.

Can you have a look at the screenshot please?
Thanks again
[
![Screenshot 2023-05-10 at 10 07 44](https://github.com/gopro/labs/assets/132943298/5c2b41a3-d36d-41d0-87bf-d9a638fdaa85)
](url)

**Reply by dnewman-gpsw** [reply]
Do not use commas or slashes. Lots of weird stuff.

The command you want is dTmTP.3600r!10:00S!17:00E!R   Or if you don't want to change the default boot mode from video (your current issue) use !10:00NmTP.3600r!17:00E!R

**Reply by motionben** [reply]
Thank you, honestly I don't know why added the commas, I was probably thinking of tags like for social media, lol.

This command worked: "dTmTP.3600r!10:00S!17:00E!R" 

The only thing is that the GoPro isn't going into sleep mode directly after the first photo, it stays on. In the preferences is a setting for "auto power off" I could set it manually to 5min. but I was wondering if there is a command for a shorter time or let's say that it turns off directly until the next interval.

Thanks again!

**Reply by dnewman-gpsw** [reply, marked as answer]
That is different type of command, what you are using is slightly more reliable.  If you wanted to turn off, you can't use a timelapse mode, you have use normal Photo like this: mPdP>10:00<16:00!3600SQ~!10:00S!1R

**Reply by motionben** [reply]
Thank you so much, I will need a few days, to check if everything works well. With the previous command the camera didn't continue the interval, it just stayed on recording with the first picture taken. 
But using the custom timelapse qr code generator it works now. I definitely will try the last command regarding sleep mode.
Thanks again

**Reply by motionben** [reply]
> That is different type of command, what you are using is slightly more reliable. If you wanted to turn off, you can't use a timelapse mode, you have use normal Photo like this: mPdP>10:00<16:00!3600SQ~!10:00S!1R

After some testing this command was the one working the best! Thank you so much!!!
